build/stf-run-ci/tasks/deploy_stf.yml

```diff
@@ -27,6 +27,14 @@
               enabled: {{ __service_telemetry_metrics_enabled }}
               storage:
                 strategy: {{ "ephemeral" if __service_telemetry_storage_ephemeral_enabled else "persistent" }}
+          logs:
+            loki:
+              enabled: {{ __service_telemetry_logs_enabled }}
+              replicationFactor: 1
+              flavor: 1x.extra-small
+              storage:
+                objectStorageSecret: test
+                storageClass: standard
```

Comment:
I think this is supposed to be `storageClassName`?

Reply:
Further review, I think this is going to be correct and that the `storage.storageClassName` parameter should change to be consistent with other `storage...` parameters.

Reply:
Interesting. I remember changing the storage class to standard2 here at some point and it seemed to work :thinking:. I'll fix it today.